models/aeif_cond_alpha.cpp

```diff
@@ -278,38 +278,50 @@ nest::aeif_cond_alpha::Parameters_::set( const DictionaryDatum& d )
 
   updateValue< double >( d, names::gsl_error_tol, gsl_error_tol );
 
-  if ( Delta_T != 0. && V_peak_ <= V_th )
-    throw BadProperty( "V_peak must be larger than threshold." );
-  else if ( Delta_T != 0. )
+  if ( V_reset_ >= V_peak_ )
+  {
+    throw BadProperty( "Ensure that: V_reset < V_peak ." );
+  }
+
+  if ( Delta_T < 0. )
+  {
+    throw BadProperty( "Delta_T must be positive." );
+  }
+  else if ( Delta_T > 0. )
   {
     // check for possible numerical overflow with the exponential divergence at
-    // spike time
-    double max_exp_arg = std::log( std::numeric_limits< double >::max() );
+    // spike time, keep a 1e20 margin for the subsequent calculations
+    const double max_exp_arg =
+      std::log( std::numeric_limits< double >::max() ) - 20.;
```

Comment:
@Silmathoron Here you mix natural and decadic log. I would suggest 
```C++
std::log( std::numeric_limits< double >::max() / 1e20 )
```
for clarity and correctness.